tests/unit/search/test_search_query.py

```diff
@@ -186,10 +221,29 @@ def test_author_colon():
     query = IQ('author: vagenas')
 
     expected = {
-        'bool': {
-            'should': [
-                {'match': {'authors.name_variations': 'vagenas'}},
-                {'match': {'authors.full_name': 'vagenas'}}
+        "bool": {
+            "should": [
+              {
+                "bool": {
+                  "should": [
+                    {
+                      "match": {
+                        "authors.name_variations": "vagenas"
+                      }
+                    },
+                    {
+                      "match": {
+                        "authors.full_name": "vagenas"
+                      }
+                    }
+                  ]
+                }
+              },
+              {
+                "match": {
+                  "authors.inspire_bai": "vagenas"
+                }
```

Comment:
I also wonder why the ordering/nesting is different here: is the query produced non-deterministic?